internal/error.go

```diff
@@ -43,71 +43,71 @@ If activity fails then *ActivityError is returned to the workflow code. The erro
 and actual error which caused activity failure. This internal error can be unwrapped using errors.Unwrap() or checked using errors.As().
 Below are the possible types of internal error:
 1) *ApplicationError: (this should be the most common one)
-	*ApplicationError can be returned in two cases:
-		- If activity implementation returns *ApplicationError by using NewApplicationError()/NewNonRetryableApplicationError() API.
-		  The error would contain a message and optional details. Workflow code could extract details to string typed variable, determine
-		  what kind of error it was, and take actions based on it. The details are encoded payload therefore, workflow code needs to know what
-          the types of the encoded details are before extracting them.
-		- If activity implementation returns errors other than from NewApplicationError() API. In this case GetOriginalType()
-		  will return original type of error represented as string. Workflow code could check this type to determine what kind of error it was
-		  and take actions based on the type. These errors are retryable by default, unless error type is specified in retry policy.
+*ApplicationError can be returned in two cases:
+- If activity implementation returns *ApplicationError by using NewApplicationError()/NewNonRetryableApplicationError() API.
```

Comment:
There appears to be some unnecessary whitespace changes

Reply:
oops, this was because I was stripping whitespace at the beginning and ending of lines, and `format.Source([]byte(newFile))` won't automatically add back spacing to comments. Fixed!